README.md

```diff
@@ -0,0 +1,133 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b> hello-react-front-end </b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 hello-react-front-end<a name="about-project"></a>
+
+**hello-react-front-end** Iis a frontend application for hello-rails-api backend.
```

Comment:
- [ ] [OPTIONAL] Kindly remove this section of the README because it has already been provided properly below.